packages/avatars/stories/examples/MenuUsage.tsx

```diff
@@ -0,0 +1,159 @@
+import { Story } from '@storybook/react';
+import {
+  Dropdown,
+  Trigger,
+  Menu,
+  MediaItem,
+  MediaFigure,
+  MediaBody,
+  ItemMeta
+} from '@zendeskgarden/react-dropdowns';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { Avatar } from '@zendeskgarden/react-avatars';
+import { Button } from '@zendeskgarden/react-buttons';
+import { PALETTE } from '@zendeskgarden/react-theming';
+
+export const MenuUsage: Story = () => {
+  const [highlightedItem, setHighlightedItem] = useState<number | null>();
+
+  return (
+    <Grid>
+      <Row>
+        <Col>
+          <Dropdown onStateChange={changes => setHighlightedItem(changes.highlightedIndex)}>
+            <Trigger>
+              <Button>Default menu</Button>
+            </Trigger>
+            <Menu hasArrow>
+              <MediaItem value="linden">
+                <MediaFigure>
+                  <Avatar
+                    size="small"
+                    status="away"
+                    surfaceColor={highlightedItem === 0 ? PALETTE.blue[100] : undefined}
+                  >
+                    <img alt="Linden" src="images/avatar-25.png" />
+                  </Avatar>
+                </MediaFigure>
+                <MediaBody>
+                  Linden
+                  <ItemMeta>linden@zendesk.garden</ItemMeta>
+                </MediaBody>
+              </MediaItem>
+              <MediaItem value="reed">
+                <MediaFigure>
+                  <Avatar
+                    size="small"
+                    status="available"
+                    surfaceColor={highlightedItem === 1 ? PALETTE.blue[100] : undefined}
+                  >
+                    <img alt="Reed" src="images/avatar-30.png" />
+                  </Avatar>
+                </MediaFigure>
+                <MediaBody>
+                  Reed
+                  <ItemMeta>reed@zendesk.garden</ItemMeta>
+                </MediaBody>
+              </MediaItem>
+              <MediaItem value="sage">
+                <MediaFigure>
+                  <Avatar
+                    size="small"
+                    badge="3"
+                    surfaceColor={highlightedItem === 2 ? PALETTE.blue[100] : undefined}
+                  >
+                    <img alt="Sage" src="images/avatar-48.png" />
+                  </Avatar>
+                </MediaFigure>
+                <MediaBody>
+                  Sage
+                  <ItemMeta>sage@zendesk.garden</ItemMeta>
+                </MediaBody>
+              </MediaItem>
+            </Menu>
+          </Dropdown>
+        </Col>
+        <Col>
+          <Dropdown onStateChange={changes => setHighlightedItem(changes.highlightedIndex)}>
+            <Trigger>
+              <Button>Small menu</Button>
+            </Trigger>
+            <Menu hasArrow isCompact>
+              <MediaItem value="clove">
+                <MediaFigure>
+                  <Avatar
+                    size="extrasmall"
```

Comment:
Per https://zendeskgarden.github.io/react-components/dropdowns/#advanced-menu-usage these should've been updated to be `extraextrasmall`. Again, there appears to be a rendering issue with storybook.

Reply:
I've updated the size and they seem to be positioned correctly for me now. What do you think?